style.css

```diff
@@ -0,0 +1,424 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+@media screen and (max-width: 768px) {
+  .header {
+    color: #fff;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
```

Comment:
- [ ] Kindly make sure your page doesn't overlap. The width is supposed to fits in the screen size.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199293492-7950ab67-13ba-4b99-937a-3527ba6d2140.png)
![image](https://user-images.githubusercontent.com/97100091/199293815-bb50db30-018f-4cfd-96f3-4f3e95fb6d78.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199293631-82b9ffda-fb65-44f5-812f-a2a27ab8deeb.png)